PWGCF/FemtoUniverse/Tasks/femtoUniversePairTaskTrackTrackMultKtExtended.cxx

```diff
@@ -65,12 +65,13 @@ struct femtoUniversePairTaskTrackTrackMultKtExtended {
 
   /// Table for both particles with separate configurables for De
   struct : o2::framework::ConfigurableGroup {
-    Configurable<float> ConfNsigmaCombined{"ConfNsigmaCombined", 3.0f, "TPC and TOF Pion Sigma (combined) for momentum > ConfTOFPtMin"};
-    Configurable<float> ConfNsigmaTPC{"ConfNsigmaTPC", 3.0f, "TPC Pion Sigma for momentum < ConfTOFPtMin"};
-    Configurable<float> ConfNsigmaTPCDe{"ConfNsigmaTPCDe", 3.0f, "TPC Deuteron Sigma for momentum < ConfTOFPtMin"};
-    Configurable<float> ConfNsigmaTOFDe{"ConfNsigmaTOFDe", 1.3f, "TOF Deuteron Sigma"};
-    Configurable<float> ConfTOFPtMin{"ConfTOFPtMin", 0.5f, "Min. Pt for which TOF is required for PID."};
-    Configurable<float> ConfTOFPtMinDe{"ConfTOFPtMinDe", 0.5f, "Min. Pt for De for which TOF is required for PID."};
+    Configurable<bool> IsKaonNsigma{"IsKaonNsigma", false, "Enable a strict cut selection for K+ and K-"};
+    Configurable<float> ConfNsigmaCombined{"ConfNsigmaCombined", 3.0f, "TPC and TOF Pion Sigma (combined) for momentum > ConfTOFpMin"};
+    Configurable<float> ConfNsigmaTPC{"ConfNsigmaTPC", 3.0f, "TPC Pion Sigma for momentum < ConfTOFpMin"};
+    Configurable<float> ConfNsigmaTPCDe{"ConfNsigmaTPCDe", 2.0f, "TPC Deuteron Sigma for momentum < ConfTOFpMin"};
+    Configurable<float> ConfNsigmaTOFDe{"ConfNsigmaTOFDe", 2.0f, "TOF Deuteron Sigma"};
+    Configurable<float> ConfTOFpMin{"ConfTOFpMin", 0.5f, "Min. Pt for which TOF is required for PID."};
```

Comment:
is it still Pt or P?

Reply:
regarding the help texts of the configurables

Reply:
Hi Maja, I approved it before seeing your message. It should be p everywhere I think

Reply:
But you are right, @a-m-andrushko please change the help text of the configurables in your next PR

Reply:
I think it would be helpful to fix the help messages, otherwise soon someone will get confused.

Reply:
I will commit a corrected version with all suggestions first thing tomorrow